pipeline/terraform/test-environment/test-environment.tf

```diff
@@ -117,6 +125,67 @@ resource "aws_ecs_task_definition" "test-environment-mhs-outbound-task" {
   execution_role_arn = var.task_execution_role
 }
 
+resource "aws_ecs_task_definition" "test-environment-mhs-spineroutelookup-task" {
+  family = "mhs-spineroutelookup-task-${var.build_id}"
+
+  volume {
+    name = "certs-volume"
+    host_path = "/home/ec2-user/certs"
+  }
+
+  container_definitions = jsonencode(
+  [
+    {
+      name = "spineroutelookup"
+      image = "${var.ecr_address}:spineroutelookup-${var.build_id}"
+      essential = true
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          awslogs-group = "/ecs/spineroutelookup-test-environment"
+          awslogs-region = var.region
+          awslogs-stream-prefix = var.build_id
+        }
+      }
+
+      mountPoints = [{
+        sourceVolume = "certs-volume"
+        containerPath = "/usr/src/app/mhs/common/data/certs/"
+      }]
+
+      environment = [
+        {
+          name = "MHS_LOG_LEVEL"
+          value = var.mhs_log_level
+        },
+        {
+          name = "MHS_SDS_URL"
+          value = var.spineroutelookup_service_sds_url
+        },
+        {
+          name = "MHS_DISABLE_SDS_TLS"
+          value = var.spineroutelookup_service_disable_sds_tls
+        }
+      ]
+
+      portMappings = [
+        {
+          containerPort = 80
+          hostPort = var.spineroutelookup_service_port
+          protocol = "tcp"
+        }
+      ]
+    }
+  ]
+  )
+  cpu = "128"
+  memory = "256"
```

Comment:
Can't seem to find the logs for this container, but I think with 4 containers at 256 memory I think that's the limit of the ecs instance, I'd maybe just double check that all 4 services are starting correctly